.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Codeowners for reviews on PRs
+
+*  @bbrown105 @dustingooding @JamesAdkison @neatudarius
```

Comment:
Please use this thread to comment if we want or not to add this files.

If yes, should we added it the standard?

If yes, people don't have pemissions - CC: @camio - `Unknown owner on line 3: make sure @neatudarius exists and has write access to the repository`.

Reply:
I think the error message is resolved now, right?

Reply:
I think bbrown105 is not added, but bretbrownjr is :D